src/wait.ts

```diff
@@ -1,15 +1,17 @@
 import {watch} from 'fs'
 
+const timeoutSeconds = 60
```

Comment:
Thank you for your contribution! 

I think the server-side timeout is 45 seconds - https://docs.saucelabs.com/secure-connections/sauce-connect/advanced/specifications/#start-timeouts

If it didn't start within 45 seconds, might as well exit or retry. Usually,, tunnel start doesn't take more than 10-15 seconds.

Reply:
Hmm I think it was taking longer for us. Maybe sometimes there’s a slow down before the server side timeout starts?

Reply:
yeah, possible. Or maybe the server timeout was extended for some short time. It happens when they introduce some new features that require a longer setup. kk, let's leave it at 60, no harm in waiting an extra 15 sec.